projects/project_ideas.md

```diff
@@ -0,0 +1,59 @@
+# CS 7290: Project Ideas
+**Prof. Robert Ness**
+
+## 1. State-of-the-art Causal Inference methods
+
+You are encouraged to explore Causal Inference literature and to explore the papers that suits your research interests. For reference, take a look at the proceedings of NeurIPS and UAI for last four years. The applications of these methods do not limit to AI and computer science. They expand to Epidemiology, Social sciences, Medicines, Statistics and Political sciences. You can pick the case studies that suit your interest.
+
+**Expectations**: Comprehensive literature survey and understanding of the problem, implementation of state-of-the-art methods, comparative analysis of similar methods and suggesting possible improvements.
+
+
+## 2. Causal Inference with Probabilistc Generative Models
+
+[https://arxiv.org/abs/1805.06826]()
+
+**Introduction**:
+
+Causal inference from observational data often assumes “ignorability,” that all confounders are observed. This assumption is standard yet untestable. However, many scientific studies involve multiple causes, different variables whose effects are simultaneously of interest. *The deconfounder*, an algorithm that combines unsupervised machine learning and predictive model checking to perform causal inference in multiple-cause settings. *The deconfounder* infers a latent variable as a substitute for unobserved confounders and then uses that substitute to perform causal inference. The deconfounder provides a checkable approach to estimating closer-to-truth causal effects.
+
+**Keywords**: Potential Outcomes, Probabilistic Generative Models.
+
+**Data**: Any dataset that comes with structured content and response. For example, Actor and movie revenue or Smoking cancer dataset (from paper), Topics (extracted from documents) and response(for example, # views of document).
+
+**Expectations**: Deconfounded causal effect. For example, What is
+the causal effect on movie revenue given that Oprah Winfrey is casted in the
+movie? What is the causal effect of adding political topics in your article to the number of views?
+
+You can go through this project **(link to preprint)** that uses variational autoencoders to measure the causal effect of latent variables on the response rate.
+
+**Point of Contact**: Prof. Robert Ness, Kaushal Paneri.
+
+## 3. Causal Implicit Generative Models (CausalGANs)
```

Comment:
Remove for now, add to a separate PR, and I will review later.  I need to have a think about this.